public/js/icinga/behavior/modal.js

```diff
@@ -36,6 +36,9 @@
      * @returns {boolean}
      */
     Modal.prototype.onModalToggleClick = function(event) {
+        // Disable pointer events to block further function calls
+        event.currentTarget.style.pointerEvents = 'none';
```

Comment:
Please move that further below. It's not part of the initialization procedures after all.